crates/languages/src/json.rs

```diff
@@ -240,7 +242,19 @@ async fn get_cached_server_binary(
     .log_err()
 }
 
-fn schema_file_match(path: &Path) -> &Path {
-    path.strip_prefix(path.parent().unwrap().parent().unwrap())
+#[inline]
+fn schema_file_match(path: &Path) -> String {
+    #[cfg(not(target_os = "windows"))]
+    return path
+        .strip_prefix(path.parent().unwrap().parent().unwrap())
         .unwrap()
+        .display()
+        .to_string();
+    #[cfg(target_os = "windows")]
+    return path
+        .strip_prefix(path.parent().unwrap().parent().unwrap())
+        .unwrap()
+        .display()
+        .to_string()
+        .replace("\\", "/");
```

Comment:
On Windows, this function results in `"Zed\settings.json"`, which is not currently supported by `fileMatch`. Therefore, it is converted to `"Zed/settings.json"`.

Reply:
Tbh we can just skip the `cfg` dance and use `.replace('\\', "/")` on both platforms. This code isn't really hot.

Reply:
Thanks for your suggestion. I've removed the `cfg`s.